src/cache/cache_las.c

```diff
@@ -1137,7 +1147,7 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
 			 */
 			if (upd_type == WT_UPDATE_BIRTHMARK &&
 			    __wt_txn_visible_all(
-			    session, las_txnid, las_timestamp) &&
+			    session, las_txnid, durable_timestamp) &&
```

Comment:
This change isn't obvious to me - can you add an assertion that `durable_timestamp != WT_TS_MAX && durable_timestamp >= las_timestamp`, that will make me more comfortable.